docs/content/preview/explore/ycql-language/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: YCQL features
+headerTitle: YCQL features
+linkTitle: YCQL features
+description: Explore core features in YCQL
+headcontent: Explore core features in YCQL
+image: /images/section_icons/api/ycql.png
+menu:
```

Comment:
Should it be 'CQL features' and 'Explore core CQL features in YCQL'?

Reply:
The section is going to be about `YCQL`. Only the `Cassandra feature support` page is about Cassandra ..

Reply:
Right, but in Explore the intention is to introduce CQL users to YugabyteDB - they don't even know what YCQL is at this point, though they might guess. I thought the intention was to mirror the 'SQL features' section. That is also about YSQL basically, but 'SQL' orients them, no?